### ISSUE TITLE
Do not exclude aria-hidden nodes in visual rules

### ISSUE THREAD

**Issue by jnurthen** [dequelabs/axe-core#170]
Nodes hidden using aria-hidden should not be excluded from the following rules 

accesskeys
blink
color-contrast
duplicate-id (technically per WCAG but unlikely to actually have an a11y impact)
marquee
video-caption
video-description

Currently isHidden excludes any nodes hidden using aria-hidden from these checks which is not valid.

See testcase for color-contrast at http://output.jsbin.com/xulehoniyu.
Both lines should fail colour contrast, but only 1 gets picked up by aXe currently as the 2nd is hidden by aria-hidden. All the other tests would fail in a similar way.

**Comment by WilcoFiers** [comment]
We have this issue resolved for our next release. Thanks for contributing!!
